### PR TITLE
RPM python 3 and github actions workflow

### PR DESCRIPTION
**Pull request by abraverm**
The RPM package refactored to be based on Python 3 virtual environment and to be build as part of Github actions, on release  it will be added as another artifact to download.

### CODE REVIEW

**Comment by abraverm** [comment]
works in my repository:
https://github.com/abraverm/linchpin/releases/tag/test6

**Comment by samvarankashyap** [comment]
@abraverm could you create a corresponding issue on github for this PR. 
before you merge this PR. 
Thanks

**Comment by abraverm** [comment]
fixes #1678

File: .github/workflows/packaging.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: "Build packages"
+
+on:
+  push:
+    branches:
+      - develop
```
Comment (14rcole) [inline .github/workflows/packaging.yml]: Are we building an RPM for every single change to the develop branch?

Comment (abraverm) [reply .github/workflows/packaging.yml]: building but not publishing, so it becomes like another test.

**Comment by abraverm** [comment]
[merge]

**Pull request merged**